.github/workflows/code-cov-report.yml

```diff
@@ -0,0 +1,49 @@
+name: Code Coverage Cron Report
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *' # Run daily at midnight UTC
+  
+jobs:
+  code_cov_report:
+    name: Code Coverage Report
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: "16.x"
+        
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: 'go.mod'
+
+      - name: Install Go dependencies
+        run: go mod download
+      
+      - name: Build
```

Comment:
probably this step is not needed, e2e just build the binary I believe so (also the binary needs to be coverage ready)